Frontend/ui-customer/src/Services/API.purs

```diff
@@ -980,6 +981,43 @@ instance showSelectListRes :: Show SelectListRes where show = genericShow
 instance decodeSelectListRes :: Decode SelectListRes where decode = defaultDecode
 instance encodeSelectListRes  :: Encode SelectListRes where encode = defaultEncode   
 
+-------------------------------------------------------------------------------Estimated Quotes With Booking API Types ------------------------------------------------------------
+data SelectListBookingReq = SelectListBookingReq String 
```

Comment:
@soumyajit2304 No need to types and instances , You can handle in the existing API itself